packages/terra-consumer-nav/src/components/nav-burger-button/NavBurgerButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import IconMenu from 'terra-icon/lib/icon/IconMenu';
+import styles from './NavBurgerButton.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  handleClick: PropTypes.func.isRequired,
+};
+
+const NavBurger = ({
+  handleClick,
+  ...customProps
+}) => (
+  <Button
+    {...customProps}
+    onClick={handleClick}
```

Comment:
question? Should we do this or is it fine if they just pass onClick into customProps?

Reply:
this would throw a prop warning to the consumer making sure they have a handleClick method. I think the terra team tries to not use prop's that are the same name as actual html props.